lib/service_extensions.dart

```diff
@@ -107,3 +107,17 @@ const slowAnimations = ServiceExtensionDescription<num>(
 // This extension should never be displayed as a button so does not need a
 // ServiceExtensionDescription object.
 const String didSendFirstFrameEvent = 'ext.flutter.didSendFirstFrameEvent';
+
+// If you add a service extension to this file whose state should be restored on
+// page refresh/initial startup, add the extension to this list.
+const List<String> statefulExtensionsWhitelist = [
```

Comment:
nit: maybe call this
toggleableExtensionsWhitelist
?
These are extensions where the value can be toggled and set. That isn't a prefect fit either but calling them stateful is a bit confusing as there could be other stateful extensions that we wouldn't want to include here unless they matched the toggleable conventions.

Reply:
done.